locations/spiders/virginia_health_inspection_blue_ridge.py

```diff
@@ -0,0 +1,55 @@
+from scrapy import Spider
+from scrapy.http import JsonRequest
+
+from locations.dict_parser import DictParser
+
+
+class VirginiaHealthInspectionBlueRidgeSpider(Spider):
+    name = "virginia_health_inspection_blue_ridge"
+    item_attributes = {}
+    start_urls = ["https://inspections.myhealthdepartment.com"]
+
+    def start_requests(self):
+        for url in self.start_urls:
+            yield JsonRequest(
+                url=url,
+                callback=self.parse_location_list,
+                data={
+                    "data": {
+                        "path": "va-blue-ridge",
+                        "programName": "",
+                        # TODO: make the date dynamic
+                        "filters": {"date": "2024-01-01 to 2024-09-27", "purpose": "", "city": "", "zip": ""},
+                        "start": 0,
+                        "count": 2000,
+                        "searchQueryOverride": None,
+                        "searchStr": "",
+                        "lat": 0,
+                        "lng": 0,
+                    },
+                    "task": "searchInspections",
+                },
+            )
+
+    def parse_location_list(self, response):
+        for inspection in response.json():
+            item = DictParser.parse(inspection)
+
+            item["ref"] = inspection["inspectionID"]
```

Comment:
If there is a OSM ref, you can also set it here. It would make conflation easier.
```suggestion
            item["ref"] = item["extras"]["ref:US:abc"] = inspection["inspectionID"]
```

Reply:
Not quite sure what this comment means, is this an example or is `ref:US:abc` something specific?

Reply:
That was a place holder. In GB we have similar food hygiene data, we tag OSM with `fhrs:id`.